docs/src/appendix/symmetric_tutorial.md

```diff
@@ -0,0 +1,1289 @@
+    tensor.
+
+
+### [Fusion trees and how to use them](@id sss_fusion_trees)
+
+This view of the underlying symmetry structure in terms of fusion trees and corresponding
+array blocks is a very convenient way of working with the `TensorMap` type. Consider a
+generic fusion tree of the form
+
+```@raw html
+<center><img src="../img/symmetric_tutorial/fusiontree.svg" alt="fusiontree" class="color-invertible" style="zoom: 170%"/></center>
+```
+
+which can be used to label a block of a `TensorMap` corresponding to a two-site operator.
+This object should actually be seen as a *pair of fusion trees*. The first member of the
+pair, related to the codomain of the `TensorMap`, is referred to as the *splitting tree* and
+encodes how the *coupled charge* $c$ splits into the *uncoupled charges* $s_1$ and $s_2$.
+The second member of the pair, related to the domain of the `TensorMap`, is referred to as
+the *fusion tree* and encodes how the uncoupled charges $f_1$ and $f_2$ fuse to the coupled
+charge $c$. Both the splitting and fusion tree can be represented as a [`FusionTree`](@ref)
+instance. You will find such a `FusionTree` has the following properties encoded into its
+fields:
+
+- `uncoupled::NTuple{N,I}`: a list of `N` uncoupled charges of type `I<:Sector`
+- `coupled::I`: a single coupled charge of type `I<:Sector`
+- `isdual::NTuple{N,Bool}`: a list of booleans indicating whether the corresponding uncoupled charge is dual
+- `innerlines::NTuple{M,I}`: a list of inner lines of type `I<:Sector` of length `M = N - 2`
+- `vertices::NTuple{L,T}`: list of fusion vertex labels of type `T` and length `L = N - 1`
+
+For our current application only `uncoupled` and `coupled` are relevant, since
+$\mathbb{Z}_2$ irreps are self-dual and have Abelian fusion rules. We will come back to
+these other properties when discussion more involved applications. Given some `TensorMap`,
+the method `TensorKit.fusiontrees(t::TensorMap)` returns an iterator over all pairs of
+splitting and fusion trees that label the blocks of `t`.
+
+We can now put this into practice by directly constructing the $ZZ$ operator in the irrep
+basis as a $\mathbb{Z}_2$-symmetric `TensorMap`. We will do this in three steps:
+
+- First we construct the physical space at each site as a $\mathbb{Z}_2$-graded vector space.
+- Then we initialize an empty `TensorMap` with the correct domain and codomain vector spaces built from the previously constructed physical space.
+- And finally we iterate over all splitting and fusion tree pairs and manually fill in the corresponding nonzero blocks of the operator.
+
+After the basis transform to the irrep basis, we can view the two-dimensional complex
+physical vector space we started with as being spanned by the trivial and sign irrep of
+$\mathbb{Z}_2$. In the language of TensorKit.jl, this can be implemented as a `Z2Space`, an
+alias for a
+$\mathbb{Z}_2$-[graded vector space](@ref GradedSpace)
```

Comment:
I would simply call this graded space here. Explicitly mentioning Z_2-graded might be confusing, because it is this way only because `Irrep[Z{2}]` is isomorphic to Z_2, but it is `Irrep[Z{2}]` that is providing the grading.

```suggestion
[graded vector space](@ref GradedSpace)
```